### DISCUSSION TITLE
Empty document already has 2 master Slides. Can I prevent that? Or delete them?

### DISCUSSION THREAD

**Discussion by raarts** [gitbrent/PptxGenJS#1088] [Q&A]
Is there a way to prevent that? I'd like to add (and manipulate) them myself.
Currently if I add a master slide, it becomes the third master slide. 
(The first two are called 'DEFAULT', and 'Office Theme'.)

**Comment by gitbrent** [comment]
Presentations require at least 1-2 masters; you cant have a slide without a masterpage.

Therefore, you'll be looking for the option to delete masterpages.
